cases/string/contains.yaml

```diff
@@ -0,0 +1,94 @@
+function: contains
+cases:
+  - group:
+      id: basic
+      description: Basic examples contains as prefix
+    args:
+      - value: 'abcdefg'
+        type: string
+      - value: 'abc'
+        type: string
+    result:
+      value: true
+      type: boolean
+  - group:
+      id: basic
+      description: Basic examples as case insensitivity
```

Comment:
After the initial definition of the group, you can just do 
```
  - group: basic
```

Reply:
fixed it